x-pack/solutions/security/plugins/security_solution/server/lib/siem_migrations/rules/task/rule_migrations_task_runner.ts

```diff
@@ -348,7 +364,24 @@ export class RuleMigrationTaskRunner {
     return this.data.rules.saveCompleted(ruleMigrationTranslated);
   }
 
-  private async saveRuleFailed(ruleMigration: StoredRuleMigration, error: Error) {
+  private async saveRuleMigrationCompleted({
+    isAborted = false,
+    error,
+  }: {
+    isAborted?: boolean;
+    error?: string;
+  }) {
+    await this.data.migrations.updateLastExecution({
+      id: this.migrationId,
+      lastExecutionParams: {
+        ended_at: new Date().toISOString(),
+        is_aborted: isAborted,
+        error: error ?? '',
```

Comment:
Can we avoid having `error: ''`? The ES `exist` operation will return them.
It would be better `error: null`, or even better if the `error` key does not exist.